alica_test_utility/CMakeLists.txt

```diff
@@ -2,21 +2,28 @@ cmake_minimum_required(VERSION 3.10.2)
 project(alica_test_utility)
 
 add_compile_options(-std=c++17)
-set(PKG_DEPS
-	alica_engine
-	alica_dummy_proxy
-)
-find_package(catkin REQUIRED ${PKG_DEPS})
 
-catkin_package(
-	INCLUDE_DIRS include
-	LIBRARIES alica_test_utility
-	CATKIN_DEPENDS ${PKG_DEPS}
-)
 
+# Set up utils to allow Colcon/catkin to find the packages
+include(GNUInstallDirs)
+include(CMakePackageConfigHelpers)
+set(ConfigPackageLocation ${CMAKE_INSTALL_LIBDIR}/${CMAKE_PROJECT_NAME}/cmake) 
+
+
+# Imports
+find_package(alica_common_config REQUIRED)
+find_package(alica_engine REQUIRED)
+find_package(alica_dummy_proxy REQUIRED)
+find_package(alica_solver_interface REQUIRED)
+
+
+# Build 
 include_directories(
 	include
-	${catkin_INCLUDE_DIRS}
+	${alica_solver_interface_INCLUDE_DIRS}
+	${alica_dummy_proxy_INCLUDE_DIRS}
```

Comment:
Removed all the ones that it let me - since alica_common_config is just headers it seems like it still needs it in the directories since it has no compiled file to link against